app/helpers/displayHelpers.php

```diff
@@ -2690,16 +2690,16 @@ function caCreateLinksFromText($pa_text, $ps_table_name, $pa_row_ids, $ps_class=
 		if (!$g_request) { return $pa_text; }
 		
 		foreach($pa_text as $vn_i => $vs_text) {
-			$vs_text = preg_replace("!([A-Za-z0-9]+)='([^']*)'!", "$1=\"$2\"", $vs_text);	
+			$vs_text = preg_replace("!([A-Za-z0-9]+)='([^']*)'!", "$1=\"$2\"", $vs_text);
 			$va_l_tags = array();
 			$o_links = $o_doc('l');
-			
+
 			foreach($o_links as $o_link) {
 				if (!$o_link) { continue; }
 				$vs_html = $o_link->html();
 				
-				$vs_content = preg_replace("!^<[^\>]+>!", "", $vs_html);
```

Comment:
There is no escaping going on
``` php
php > echo "!^<[^>]+>!";
!^<[^>]+>!
php > echo "!^<[^\>]+>!";
!^<[^\>]+>!
```

These are two different regexes which will behave differently, so are you sure the removal of the `\` is intentional?